tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "pretty": true,
     "target": "es5",
     "outDir": "distribution",
-    "lib": ["es2017"]
+    "lib": ["dom", "es2017"]
```

Comment:
not super pleased on this, but typedoc depends on highlightjs which depends on the dom.